tsconfig.json

```diff
@@ -1,8 +1,29 @@
 {
   "compilerOptions": {
+    "target": "es2017",
+    "module": "commonjs",
+    "declaration": true,
+    "noImplicitAny": true,
+    "removeComments": true,
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "inlineSources": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
     "baseUrl": ".",
     "paths": {
-      "@interledger/*" : ["node_modules/@interledger/*/src"],
+      "@interledger/codecs-oer" : ["packages/codecs/oer/src"],
```

Comment:
These could probably be generated using the `npx lerna ls` command ...

Reply:
Yeh, I don't expect we'll have a hundreds of packages so the wildcards are probably not necessary. We should just name all the packages explicitly